beacon_chain/types/src/beacon_block.rs

```diff
@@ -0,0 +1,150 @@
+use super::Hash256;
+use super::attestation_record::AttestationRecord;
+use super::special_record::SpecialRecord;
+use super::ssz::{
+    Encodable,
+    Decodable,
+    DecodeError,
+    SszStream,
+};
+
+pub const MIN_SSZ_BLOCK_LENGTH: usize = {
+    8 +                 // slot
```

Comment:
Why not do something like `const SLOT: usize = 8;` for all the fields?

Reply:
Implemented. Great point @HCastano